--- a/drivers/mtd/spi-nor/spi-nor.c	2019-07-30 19:04:15.000000000 -0700
+++ b/drivers/mtd/spi-nor/spi-nor.c	2020-05-18 18:15:43.425404994 -0700
@@ -2112,6 +2112,11 @@
 			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB)
 	},
 	{
+		"xt25f128b", INFO(0x0b4018, 0, 64 * 1024, 256,
+			SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
+			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB)
+	},
+	{
 		"w25q128jv", INFO(0xef7018, 0, 64 * 1024, 256,
 			SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
 			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB)
